clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -235,17 +235,20 @@ def _get_pods_info(
             "exclude_daemonset_pods",
             default=staticconf.read_bool("exclude_daemonset_pods", default=False),
         )
-        all_pods = self._core_api.list_pod_for_all_namespaces().items
-        for pod in all_pods:
-            if self._pod_belongs_to_pool(pod):
-                if exclude_daemonset_pods and self._pod_belongs_to_daemonset(pod):
-                    excluded_pods_by_ip[pod.status.host_ip].append(pod)
-                elif pod.status.phase == "Running" or self._is_recently_scheduled(pod):
-                    pods_by_ip[pod.status.host_ip].append(pod)
-                elif self._is_unschedulable(pod):
-                    unschedulable_pods.append(pod)
-                else:
-                    logger.info(f"Skipping {pod.metadata.name} pod ({pod.status.phase})")
+        label_selector = "{}={}".format(self.pool_label_key, self.pool)
+
+        for pod in self._core_api.list_pod_for_all_namespaces(label_selector=label_selector).items:
+            if pod.status.phase not in KUBERNETES_SCHEDULED_PHASES:
```

Comment:
what's this change? seems unrelated to the bug/commit

Reply:
This change is related to the changes made by @ilkinmammadzada here - https://github.com/Yelp/clusterman/pull/173/files since they were done in the same file I had to add them for fixing any merge conflicts @Rob-Johnson

Reply:
hmm, okay. maybe you just need to rebase your changes onto master?

we don't want to pollute this change with unrelated changes - it'll be hard to pull apart what's going on if this fails, and makes things more difficult if we need to revert.

so if you need to make this change, can you create a second commit + PR for review separate to the functionality change for the filtering?